include/zephyr/dt-bindings/pinctrl/npcm-pinctrl.h

```diff
@@ -0,0 +1,536 @@
+/*
+ * Copyright (c) 2024 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_PINCTRL_NPCM_PINCTRL_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_PINCTRL_NPCM_PINCTRL_H_
+
+/* Pinctrl node types in NPCM series */
+#define NPCM_PINCTRL_TYPE_PERIPH_PINMUX 0
+#define NPCM_PINCTRL_TYPE_PERIPH_PUPD   1
+#define NPCM_PINCTRL_TYPE_DEVICE_CTRL   2
+#define NPCM_PINCTRL_TYPE_LVOL          3
+
+/* Supported IO voltage level in NPCM series */
+#define NPCM_IO_VOLT_LV_3V3 0
+#define NPCM_IO_VOLT_LV_1V8 1
+
+/* Supported IO bias type in NPCM series */
+#define NPCM_BIAS_TYPE_NONE      0
+#define NPCM_BIAS_TYPE_PULL_UP   (1 << 0)
+#define NPCM_BIAS_TYPE_PULL_DOWN (1 << 1)
```

Comment:
how is this used in DT? this file must **only** contain bits used in DT.

Reply:
Fix and correct description.